kubetest/kops.go

```diff
@@ -196,7 +197,33 @@ func (k kops) IsUp() error {
 }
 
 func (k kops) DumpClusterLogs(localPath, gcsPath string) error {
```

Comment:
Can we use this for other deployments like bash and gke?

Reply:
Yes, but I figured I would break & fix kops first ;-)